### PR TITLE
debug: impl raft_log

### PR DESCRIPTION
**Pull request by overvenus**
This PR impls Debug::raft_log.

### CODE REVIEW

**Comment by overvenus** [comment]
PTAL

**Comment by siddontang** [comment]
LGTM

**Comment by hicqu** [comment]
LGTM.

File: src/raftstore/store/debug.rs [hunk 1]
```diff
@@ -60,11 +61,27 @@ impl Debugger {
...
             )),
             Err(e) => Err(box_err!(e)),
         }
     }
+
+    pub fn raft_log(&self, region_id: u64, log_index: u64) -> Result<eraftpb::Entry> {
+        let key = keys::raft_log_key(region_id, log_index);
+        match self.engines
+            .raft_engine
+            .get_msg::<eraftpb::Entry>(key.as_slice())
```
Comment (BusyJay) [inline src/raftstore/store/debug.rs]: Is `eraftpb::Entry` necessary?

File: src/raftstore/store/debug.rs [hunk 2]
```diff
@@ -60,11 +61,24 @@ impl Debugger {
...
-                format!("get none value for key {:?}", key,),
+                format!("value for key {:?} in db {:?}", key, db),
             )),
             Err(e) => Err(box_err!(e)),
         }
     }
+
+    pub fn raft_log(&self, region_id: u64, log_index: u64) -> Result<eraftpb::Entry> {
+        let key = keys::raft_log_key(region_id, log_index);
+        match self.engines.raft_engine.get_msg(key.as_slice()) {
```
Comment (BusyJay) [inline src/raftstore/store/debug.rs]: Does `&key` work?

**Pull request merged**